lib/inflow_web/router.ex

```diff
@@ -20,7 +21,8 @@ defmodule InflowWeb.Router do
   scope "/", InflowWeb do
     pipe_through [:browser, :authenticated]
 
-    get "/", PageController, :index
+    live "/", DashboardView, session: [:access_token]
```

Comment:
creates a new liveview page mounted on / and passes current session's access_token to the live view.